data/index.html

```diff
@@ -133,41 +133,59 @@ <h5 class="text-center">USGS Topo Maps</h5>
             </a>
         </div>
     </div>
-    <div class="grid__col section alternate">
-        <h2>State Geographic Information Database (SGID)</h2>
+    <div class="grid__col section">
+        <h2>Welcome to the State Geographic Information Database (SGID)</h2>
         <p class="readable">Established by state law in 1991, Utah’s SGID ensures that GIS map layers are developed in a coordinated fashion and shared openly. The SGID provides a one-stop access point to hundreds of data layers that were developed, aggregated, or acquired by state government. To the credit of the GIS community across the state, the SGID's data resources are among the most mature and extensive in the nation. </p>
-        <p class="readable">Today, interoperable GIS server technology has diminished the importance of a completely centralized GIS data repository, and it is critical that the latest, best quality GIS data resources are readily discovered and accessed. AGRC strives to ensure that this is true for SGID data layers that are available as downloadable shapefiles or live streaming data services. </p>
+        <p class="readable">The SGID was originally conceived as a single, centralized database; however, the rapid growth of GIS data across the state and the spread of cloud-based web services have made this impractical. Today, the SGID lives on in several forms: the curated Open SGID database, ArcGIS Online hosted feature services, our SGID Open Data site, and the Raster Discovery web app. Tying all these together is the SGID Index, which makes it easy to find the latest and highest quality GIS data resources no matter where they're hosted.</p>
+        <p class="readable">We have organized the SGID datasets into different categories based on some ISO thingee to bring order to the <s>galaxy</s> SGID. These categories are represented by a unified set of icons across all the SGID's different forms.</p>
```

Comment:
Can anyone shed some light on where the categories come from?

Reply:
@brigsz, @bert-granberg, or @BGranberg would know best.